jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735PersonalDeviceUsageState.java

```diff
@@ -0,0 +1,9 @@
+package us.dot.its.jpo.ode.plugin.j2735;
+
+public enum J2735PersonalDeviceUsageState {
+    unavailable,
+    aPEDESTRIAN,
+    aPEDALCYCLIST,
+    aPUBLICSAFETYWORKER,
+    anANIMAL
```

Comment:
Are these the correct values for this enum? Looks like the same ones as the J2735PersonalDeviceUserType enum.

Reply:
Good catch yeah I'll fix that.

Reply:
Thanks!